docs/fraud-proofs.md

```diff
@@ -0,0 +1,63 @@
+# QGB fraud proofs
+
+A Celestium is an L2 rollup that uses Celestia for data availability but settles on Ethereum or any other EVM L1 chain. In simpler terms, it's a layer 2 solution that leverages the Celestia blockchain security to ensure that data is available, while the actual transactions and smart contract interactions are settled on the Ethereum blockchain or any other EVM-compatible layer 1 blockchain.
+
+Celestiums use the Quantum-Gravity-Bridge (QGB) contract, deployed on an EVM chain, to verify that its transaction data has been published to Celestia correctly. And would rely on fraud proofs otherwise.
+
+# Fraud proofs
+
+Fraud proofs is the mechanism used to inform light clients in the case of an invalid rollup state transition or unavailable rollup block data. They rely on rollup full nodes getting the data that was published to Celestia, and executing all the state transitions to verify the rollup state. If they discover an invalid state transition or unavailable rollup data, they emit a fraud proof with the necessary information to convince light clients that fraud happened. This allows for trust minimized light clients, as the network only needs one honest full node to create the fraud proof and propagate it.
+
+If the Celestium is settlement smart contract based, then the contract would only need to receive a fraud proof to decide whether data was published correctly or not.
+
+## Rollup header
+
+Rollups can adopt many approaches to prove that fraud happened. One of which could be having the following fields in the rollup header:
+
+- Rollup block state root
+- A sequence of spans in Celestia: which references where the rollup data was published in the Celestia chain.
+
+Note: The sequence of spans can be defined using the following: `Height`, `start index`, and `length` in the Celestia block, in the case of a single Celestia block. However, it could be generalized to span over multiple blocks.
```

Comment:
https://github.com/orgs/community/discussions/16925

```suggestion
> [!NOTE]  
> The sequence of spans can be defined using the following: `Height`, `start index`, and `length` in the Celestia block, in the case of a single Celestia block. However, it could be generalized to span over multiple blocks.
```